vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -196,6 +196,7 @@ public class DateTimePicker
      */
     public DateTimePicker() {
         this((LocalDateTime) null);
+        setSynchronizedEvent("change");
```

Comment:
This should be set in the [root constructor](https://github.com/vaadin/flow-components/blob/main/vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java#L236) to make sure all constructor overloads are covered.

Reply:
Good catch, my mistake. Updated the PR.